importsrv/server.go

```diff
@@ -31,14 +31,19 @@ type MetricIngester interface {
 	IngestMetrics([]*metricpb.Metric)
 }
 
+type IngesterSet struct {
+	Ingesters []MetricIngester
+	Name      string
```

Comment:
We add Metric Computation Routing config name here for logging purposes. If there is a cleaner way to write this, I'm open to improvements.

Reply:
I upgraded `Name` here to a full fledged `ComputationRoutingConfig` struct, as we needed to actually apply the `MatcherConfig` rules within the GRPC server.